bindsnet/network/network.py

```diff
@@ -337,9 +337,9 @@ def run(self, inpts: Dict[str, torch.Tensor], time: int, **kwargs) -> None:
                 clamp = clamps.get(l, None)
                 if clamp is not None:
                     if clamp.ndimension() == 1:
-                        self.layers[l].s[clamp] = 1
+                        self.layers[l].s[0, clamp] = 1
```

Comment:
Should be `self.layers[l].s[:, clamp] = 1` to clamp across the minibatch dimension.

Reply:
yes, you are right.